toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -215,9 +215,14 @@ Examples:
     cache store gems-$SEMAPHORE_GIT_BRANCH-revision-$(checksum Gemfile.lock) vendor/bundle
 
 Archives file or directory specified by path and associates it with key.
```

Comment:
I would rephrase it as:

The `cache store` command archives files and/or directories specified by `path` and associates them with the given `key` value.